manual/import-tool/src/docs/ops/import-tool.asciidoc

```diff
@@ -7,7 +7,7 @@ This chapter explains how to use the tool, format the input data and concludes w
 
```

Comment:
The import documentation is branched downstream. For this particular case, I think the best thing is to just apply the relevant changes upstream and create a card on our wall to make sure they are accounted for downstream.